microsoft-edge/webdriver-chromium/index.md

```diff
@@ -366,7 +366,7 @@ EdgeDriver driver = new EdgeDriver(options);
 const edge = require('selenium-webdriver/edge');
 
 let options = new edge.Options();
-options.addArguments("headless");
+options.addArguments("--headless=new");
```

Comment:
https://www.selenium.dev/blog/2023/headless-is-going-away/#how-can-i-set-headless-mode-from-now-on
shows different code, for JS only:
![image](https://github.com/user-attachments/assets/d1ce8b39-babb-48db-93a5-26d104b9c839)

Reply:
https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/examples%2Fjavascript%2Ftest%2Fbrowser%2FedgeSpecificCaps.spec.js#L12

Indeed it is.

Do you want me to change this?

Reply:
No need to change, the current JS codes to apply `--headless=new` worked.

Reply:
closing.  thx for testing @guangyuexu